airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/AirbyteInternal.py

```diff
@@ -1,7 +1,3 @@
-#
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
```

Comment:
@alafanechere Ive never asked, what is the cannonical way to add these?

Reply:
I think the formatter adds them? But the formatter only formats metadata ci code when it runs on other steps that usually get skipped when running airbyte-ci only

Reply:
how'd they end up removed?

Reply:
When you regenerate the models it doesnt have any context about modifications we made after the last generation.

So essentially it removes the copyright